test/integration/jest.config.json

```diff
@@ -0,0 +1,16 @@
+{
+	"modulePaths": [
+		"<rootDir>/test/",
+		"<rootDir>/server/",
+		"<rootDir>/client/",
+		"<rootDir>/client/extensions/"
+	],
+	"rootDir": "./../../",
+	"testEnvironment": "node",
+	"testMatch": [
+		"**/bin/**/__integration__/*.js",
+		"**/client/**/__integration__/*.js",
+		"**/server/**/__integration__/*.js"
```

Comment:
Yes, they use __tests__ folders. I decided to follow this convention. We can easily change it to something else. There are no constraints. I thought it would be easier to migrate client tests to Jest if we move all that pass (over 500 test files) to __tests__ folder and maintain two runners for a couple of weeks so people could use both ways in this transition period. I should have more ideas to share next week :)